spec/models/course/assessment/assessment_ability_spec.rb

```diff
@@ -35,6 +44,8 @@
       it { is_expected.to be_able_to(:create, attempting_submission) }
       it { is_expected.to be_able_to(:update, attempting_submission) }
       it { is_expected.to be_able_to(:read, submitted_submission) }
+      it { is_expected.not_to be_able_to(:update, coursemate_attempting_submission) }
```

Comment:
Need to check for `:read, coursemate_attempting_submission`? Or is that covered by `:update` already?